### PR TITLE
f2f Paris biotools submit

### PR DESCRIPTION
**Pull request by redmitry**
It's still not completely biotoolsSchema 3.2 adapted

### CODE REVIEW

**Comment by bgruening** [comment]
Thats why I do not like json - one line giant monster, its like a blob ... ;)

Could we at least reformat it in a way that it is readable?

**Comment by redmitry** [comment]
formatted

**Comment by bgruening** [comment]
Merci!

**Comment by hansioan** [comment]
@redmitry @bgruening 
The JSON formatting of the tools is wrong.
bio.tools JSON formatted tools don't have properties like "summary" and "labels"

See:
https://biotools.readthedocs.io/en/latest/api_usage_guide.html#json
and another example from bio.tools:
https://bio.tools/api/jaspar
or
https://bio.tools/api/jaspar?format=json
for how JSON tools are

**Comment by redmitry** [comment]
biotoolsSchema (XML) does have the "summary". 
biotools Json schema represents the bio.tools REST API (which doesn't follow the biotoolsSchema).
The uploaded data follows:
https://gitlab.bsc.es/inb/elixir/tools-platform/biotools-schema-model/blob/master/json/biotools-3.2.0.json
which is "one-to-one" translation from the biotoolsSchema (XML) to the Json Schema.

**Comment by hansioan** [comment]
Yes but this JSON schema to which you're pointing is not used in bio.tools, thus it cannot be in this repo. 

If you want to use a JSON schema, then use:
https://github.com/bio-tools/biotoolsSchemaj

**Comment by redmitry** [comment]
There is one problem here, because the json schema in this repo is not the biotoolsSchema developed and discussed by the community, but the internal bio.tools data format. It is bio.toos that must follow biotoolsSchema. Who is in charge for the changing of bio.tools internal formats? Is this ok to maintain two different schemas?

**Comment by hansioan** [comment]
biotoolsSchema is an XML schema, not a JSON schema. I am not aware of any biotools JSON schema, aside from:
https://github.com/bio-tools/biotoolsSchemaj
which was created not long ago.
The bio.tools internal JSON rendering IS the JSON schema, any other schemas won't validate in bio.tools, thus , in my view, have no practical application.

**Comment by redmitry** [comment]
So the point that there are two different schemas biotoolsSchema (XML) and biotoolsSchema (Json) which are different - one was promoted as a "standard" while other one was never standardized and used internally by biotools.  Years ago bio.tools promised to align to the biotoolsSchema.
Three years ago I proposed to develop formal Json schema
https://github.com/bio-tools/biotoolsSchema/blob/master/versions/biotools-2.0.0/json/biotools-2.0.0.json
But received no comments since then.
My point that internal bio.tools format shouldn't be pushed as standard, especially taking into account that bio.tools changes this arbitrary.

**Comment by bgruening** [comment]
@hansioan as far as I'm concerned this does not matter much. Its a separate file in the repo, it is __not__ the bio.tools json file. My idea is to get the data here together, everyone gets its own file at the beginning. And then we can try to align all files, enhance schemas, or whatever is needed. But as soon as we have everything here, we can try to migrate the "views".

**Comment by hansioan** [comment]
@bgruening I just saw now that the files have a .opeb.json extension, so they are not bio.tools files, which is fine, I just looked at the content of these files and thought it would be a bio.tools file. I know there are also .oeb.json files which are for OpenEBench.
@redmitry Sorry for the confusion I thought you wanted to update the bio.tools files. BTW what are these .opeb.json files used for?

**Comment by bgruening** [comment]
> .opeb.json

is an interesting acronym for openebench :)

**Comment by redmitry** [comment]
@hansioan these are files taken from the bio.tools and reformatted to "supposed" biotoolsSchema.

**Comment by hansioan** [comment]
@bgruening there are .oeb.json and .opeb.json files in the data. That's why I was asking.

@redmitry Do you use these files in OpenEBench?

**Comment by redmitry** [comment]
xxx.oeb.json was some OpenEBench metrics data.
xxx.opeb.json are the bio.tools data converted into the "biotoolsSchema" format.

**Comment by hansioan** [comment]
And the xxx.opeb.json is used in OpenEBench right? Because otherwise it would not make sense to have them.

**Comment by redmitry** [comment]
No.
OEB uses its own schema, but I thought its better to use biotoolsSchema instead.
Does it make sense to push oeb raw data? 
as defined in: https://openebench.bsc.es/monitor/tool/tool.json
I am against to expose internal formats (either oeb or bio.tools).

**Comment by hansioan** [comment]
I would say yes, this is at least what I plan on having for bio.tools data. I don't see this data as "raw data". 
Regarding the oeb raw data, it is up to you to decide what formatting you want to use. I have nothing against any type of data, as long as it's used somewhere.

File: data/-cnn/-cnn.opeb.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (bgruening) [inline data/-cnn/-cnn.opeb.json]: this ID looks very strange ....

**Comment by matuskalas** [comment]
Valid discussion. (But) can we merge now? ;-)

**Pull request merged**